test/validation/api/pktio/pktio.c

```diff
@@ -2578,6 +2578,202 @@ static void pktio_test_extra_stats(void)
 	CU_ASSERT(odp_pktio_close(pktio) == 0);
 }
 
+/* Per packet protocol statistics are platform specific,
+ * skip pktio tests when underline platform not supports
```

Comment:
This comment has a few language problems. I suggest just removing it as the code is quite obvious anyway.

Reply:
ACK